host/xtest/regression_1000.c

```diff
@@ -2570,6 +2570,11 @@ static void xtest_tee_test_1034(ADBG_Case_t *c)
 
 	res = xtest_teec_open_session(&session, &large_ta_uuid, NULL,
 				      &ret_orig);
+	if (res == TEEC_ERROR_ITEM_NOT_FOUND) {
+		Do_ADBG_Log("  skip test, large TA not found");
+		return;
```

Comment:
I think the test should still log an error when the TA is not found. I suggest this:
```C
	if (res == TEEC_ERROR_OUT_OF_MEMORY) {
		Do_ADBG_Log("TEEC_ERROR_OUT_OF_MEMORY - ignored");
		return;
	}
	ADBG_EXPECT_TEEC_SUCCESS(c, res);
	if (!res)
		TEEC_CloseSession(&session);
```

Reply:
Thanks your review and suggestion. I've changed it and run xtest test result is expected. Please help to review and merge it if this is valuable. Thanks for your time on this.
However, seems I didn't know why I create another PR for this issue. I will close this PR and need your help to check another one. Sorry and Thanks.